pkg/cmd/roachprod/flags.go

```diff
@@ -184,7 +184,7 @@ func initFlags() {
 		"lifetime", "l", 12*time.Hour, "Lifetime of the cluster")
 
 	listCmd.Flags().BoolVarP(&listCost,
-		"cost", "c", os.Getenv("ROACHPROD_NO_COST_ESTIMATES") != "true",
+		"cost", "c", false,
```

Comment:
What about setting the default to `os.Getenv("ROACHPROD_COST_ESTIMATES") == "true"`? I'm personally down for this change, but in case someone does like seeing the costs.